Dockerfile

```diff
@@ -69,12 +69,12 @@ ADD ./custom.css /home/$NB_USER/.jupyter/custom/custom.css
 ADD ./default_notebook.ipynb /home/$NB_USER/notebook.ipynb
 
 # Download notebooks
-RUN cd   /home/$NB_USER/;  \
-    wget -O Source-main.zip https://github.com/fair-ease/Source/archive/refs/heads/main.zip; unzip Source-main.zip; \
-    rm /home/$NB_USER/Source-main.zip
-
-RUN mv /home/$NB_USER/Source-main/notebooks /home/$NB_USER
-RUN rm -r /home/$NB_USER/Source-main
+RUN cd /home/$NB_USER/ &&  \
+    wget -O Source-main.zip https://github.com/fair-ease/Source/archive/refs/heads/main.zip && \
+    unzip Source-main.zip && \
+    rm /home/$NB_USER/Source-main.zip \
+    mv /home/$NB_USER/Source-main/notebooks /home/$NB_USER \
+    rm -r /home/$NB_USER/Source-main
```

Comment:
See my above comment. Cleaning up in a commit does not reduce the image size when you added the files in a previous RUN command. So aggregating them in one RUN command will reduce the image size.